backend/internal/storage/storage.go

```diff
@@ -60,15 +60,25 @@ func New(cfg *config.Config, s3 *storage.S3, metrics *monitoring.Metrics) (*Stor
 	}, nil
 }
 
-func (s *Storage) UploadKey(key string, retryCount int) error {
+func (s *Storage) UploadSessionFiles(sessID uint64) error {
+	sidStr := strconv.FormatUint(sessID, 10)
+	if err := s.uploadKey(sessID, sidStr+"/dom.mob", 5); err != nil {
+		return err
+	}
+	if err := s.uploadKey(sessID, sidStr+"/devtools.mob", 5); err != nil {
```

Comment:
Are you sure we should split devtools.mob into 2 files: devtools+s and devtools+e ?

Reply:
Actually, this is what I am hesitating about. On one hand it makes sense, because  beginning of a session gets downloaded fast, so maybe logs shuld behave the same way?